File: sample-app/src/main/java/io/github/lazymockbean/sampleapp/controller/ExampleController.java
package io.github.lazymockbean.sampleapp.controller;

import io.github.lazymockbean.sampleapp.service.ExampleService;
import org.springframework.stereotype.Controller;

@Controller
public class ExampleController {
    private final ExampleService exampleService;

    public ExampleController(ExampleService exampleService) {
        this.exampleService = exampleService;
    }

    public String getExampleA1() {
        return exampleService.getExample1();
    }

    public String getExampleA2() {
        return exampleService.getExample2();
    }
}

File: sample-app/src/main/kotlin/io/github/taesk/lazymockbean/sampleapp/controller/ExampleController.kt
package io.github.taesk.lazymockbean.sampleapp.controller

import io.github.taesk.lazymockbean.sampleapp.service.ExampleService
import org.springframework.stereotype.Controller

@Controller
class ExampleController(
    private val exampleService: ExampleService
) {
    fun getExampleA1(): String {
        return exampleService.getExample1()
    }

    fun getExampleA2(): String {
        return exampleService.getExample2()
    }
}
